diff --git a/dist/node/constants.js b/dist/node/constants.js
index 334b9a0929295a440030a8bdb383c56a322e4301..7d133d3d87e9541508ce875a5656be22f430ea03 100644
--- a/dist/node/constants.js
+++ b/dist/node/constants.js
@@ -107,6 +107,7 @@ const KNOWN_ASSET_TYPES = [
     'webmanifest',
     'pdf',
     'txt',
+    'md',
 ];
 const DEFAULT_ASSETS_RE = new RegExp(`\\.(` + KNOWN_ASSET_TYPES.join('|') + `)(\\?.*)?$`);
 const DEP_VERSION_RE = /[?&](v=[\w.-]+)\b/;
